src/src/pages/capabilities/MyCapabilities.js

```diff
@@ -68,6 +70,89 @@ export default function MyCapabilities() {
   useEffect(() => {
     setShowAwsResourcesSpinner(isLoadingAwsResourcesCounts);
   }, [isLoadingAwsResourcesCounts]);
+
+
+  const columns = useMemo(
```

Comment:
This is not something we can do anything about and the `useMemo` is part of the official documentation too.
It just feels funny, that we are passing something a function and then insisting on the result never changing.
Why have the function in the first place 🤔 
Oy vey; I am probably just missing some fascinating big-picture things